### PR TITLE
Automatically update any measures on deploy

### LINKED ISSUE #1517: Automate measures deployment
* Fab command to deploy a measure in staging or production
  * This should also remember to clear cloudflare cache
* Bot command to do same
* Logic in op bot to detect if there are measure changes in a PR, and if so, to trigger a measure build after the next successful deploy

### PR DESCRIPTION
**Pull request by sebbacon**
Closes #1517

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -168,6 +168,33 @@ def run_migrations():
...
+def build_measures(environment=None, measures=None):
+    env.app = environments[environment]
+    env.environment = environment
+    env.path = "/webapps/%s" % env.app
+
+    with cd(env.path):
+        with prefix('source .venv/bin/activate'):
+            run("cd openprescribing/ && "
+                "python manage.py import_measures --measure {}".format(
+                    measures))
```
Comment (inglesp) [inline fabfile.py]: If we're going to allow this to be called via ebmbot, it'd be nice to be able to rebuild all measures.

File: fabfile.py [hunk 2]
```diff
@@ -168,6 +168,33 @@ def run_migrations():
         warn("Refusing to run migrations in staging environment")
 
 
+@task
+def build_measures(environment=None, measures=None):
+    env.app = environments[environment]
+    env.environment = environment
+    env.path = "/webapps/%s" % env.app
```
Comment (inglesp) [inline fabfile.py]: Could you extract this code to set up the environment as its own function, which you could then also call from `deploy()`?

And while I'm looking at this -- should `graceful_reload` also take an environment?

File: fabfile.py (hunk 1 above)
Comment (sebbacon) [reply fabfile.py]: I wasn't going to allow it to be called via ebmbot, on the grounds that you shouldn't need to rebuild any measures that haven't changed....?

File: fabfile.py (hunk 2 above)
Comment (sebbacon) [reply fabfile.py]: 1. Yes
2. No - I see no benefit to it being its own fab command so instead think we should remove the `@task`

File: fabfile.py (hunk 1 above)
Comment (inglesp) [reply fabfile.py]: I was thinking of the case when `import_measures` ran incorrectly... but this hasn't happened for a long time and in any case we could run this from the console.  So scratch this suggestion.

**Pull request merged**